src/attributes.jl

```diff
@@ -1856,6 +1856,86 @@ function attribute_value_type(::ConstraintConflictStatus)
     return ConflictParticipationStatusCode
 end
 
+"""
+    UserDefinedFunction(name::Symbol, arity::Int) <: AbstractModelAttribute
+
+Set a user-defined function by the name of `name` with `arity` arguments.
+
+The value to be set is a tuple containg one to three functions, representing the
+zero, first, and second-order derivatives of the function.
```

Comment:
zeroth? or evaluation and first two derivatives? or just remove the fragment after "representing the ..." because it is redundant/repetitive with what's immediately below.